DEVELOPMENT.md

```diff
@@ -168,13 +169,18 @@ Next, run:
 ko apply -f config/
 ```
 
+The above step is equivalent to applying the `serving.yaml` for released versions of
+Knative Serving.
+
 You can see things running with:
 
 ```shell
```

Comment:
it seems in other places it's marked as ````console`
But just a suggestion, rendering probably is the same

Reply:
Looks like console is an alias for shell session. Given that this is a command + output from the command I am going to update it.

https://github.com/github/linguist/blob/master/lib/linguist/languages.yml

```suggestion
```console
```